doc_source/cli-configure-completion.md

```diff
@@ -141,11 +134,11 @@ For the AWS completer to work successfully, you must first add it to your comput
 
 ### Enable command completion<a name="cli-command-completion-enable"></a>
 
-To enable command complettion, run the command for the shell that you're using\. You can add the command to your shell's RC file to run it each time you open a new shell\. In each command, replace the path `/usr/local/aws/bin` with the one found on your system in the previous section\.
+To enable command complettion, run the command for the shell that you're using\. You can add the command to your shell's RC file to run it each time you open a new shell\. In each command, replace the path `/usr/local/bin` with the one found on your system in the previous section\.
 + **`bash`** – Use the built\-in command `complete`\.
 
   ```
-  $ complete -C '/usr/local/aws/bin/aws_completer' aws
+  $ complete -C '/usr/local/bin/aws_completer' aws
   ```
 
   Add the command to `~/.bashrc` to run it each time you open a new shell\. Your `~/.bash_profile` should source `~/.bashrc` to ensure that the command is also run in login shells\.
```

Comment:
Slight inconsistent writing style

Suggests a step instead of presenting a direct line of code.


Comparing this
>
>Reload the profile into the current session to put those changes into effect. Replace .bash_profile with the name of the shell script you discovered in the first section.
>`$ source ~/.bash_profile`

with this

>`$ complete -C '/usr/local/aws/bin/aws_completer' aws`
>Add the command to ~/.bashrc to run it each time you open a new shell. Your ~/.bash_profile should source ~/.bashrc to ensure that the command is also run in login shells.


Possible duplicate statement trailing over from the previous step? How might we consolidate this into a single previous step? (or make the text more consistent overall to use similar terminology)
 
e.g. Recommendation: Present the instruction similar to the export path step. The current export path step generally refers to the profile or rc script to append to; however, the current 'Enable command completion' steps use conflicting language that appear to semantically represent the same kind of "rc" files

source ~/.bash_profile
Add the command to ~/.bashrc to run it each time you open a new shell. Your ~/.bash_profile should source ~/.bashrc to ensure that the command is also run in login shells.